system/doc/top/templates/index.html.src

```diff
@@ -200,14 +199,12 @@ module and function name completion (tab) if the module is loaded.
 <p>
 
 <li>OpenSource users can ask questions
-and share experiences on the <a href="http://www.erlang.org/static/doc/mailinglist.html">
-Erlang questions mailing list</a>.
+and share experiences on <a href="http://erlangforums.com">
```

Comment:
Could you change it to `https` ?

Here, and pretty much everywhere else.

Reply:
will do that and more changes before OTP 26, thanks for the suggestion

Reply:
Closes #5048 as well